internal/serve/httphandler/disbursement_handler.go

```diff
@@ -276,7 +278,22 @@ func (d DisbursementHandler) GetDisbursement(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
-	httpjson.Render(w, disbursement, httpjson.JSON)
+	disbursementManagementService := services.NewDisbursementManagementService(d.Models, d.DBConnectionPool, d.AuthManager)
+	response, err := disbursementManagementService.AppendUserMetadata(ctx, []*data.Disbursement{disbursement})
```

Comment:
not sure if it's worth creating another method within the service with a signature that takes in a single disbursement instead